### PR TITLE
[stable8] 14719 without public API

### PR DESCRIPTION
**Pull request by LukasReschke**
@PVince81 @karlitschek THX. - Will also fix failing OCI unit test.

### CODE REVIEW

**Comment by PVince81** [comment]
Works as before :+1:

**Comment by nickvergessen** [comment]
works as advertised

**Comment by karlitschek** [comment]
thanks guys.

File: lib/public/iconfig.php [hunk 1]
```diff
@@ -110,16 +110,6 @@ public function deleteAppValue($appName, $key);
...
 
 	/**
-	 * Determines the apps that have the set the value for the specified
-	 * keys
-	 *
-	 * @param string $value the value to get the app for
-	 * @param string $key the key too lookup
-	 * @return array of app IDs
-	 */
-	public function getAppsForKeyValue($key, $value);
```
Comment (DeepDiver1975) [inline lib/public/iconfig.php]: Thanks a lot for breaking the public api!

Comment (DeepDiver1975) [inline lib/public/iconfig.php]: aha - introduced in #14719 and now killed - okay.

Comment (LukasReschke) [inline lib/public/iconfig.php]: First reading before blaming ;-)

Comment (DeepDiver1975) [inline lib/public/iconfig.php]: imsorry

**Pull request merged**